lib/open_food_network/order_and_distributor_report.rb

```diff
@@ -34,7 +34,9 @@ def header
     end
 
     def search
-      @permissions.visible_orders.complete.not_state(:canceled).search(@params[:q])
+      @permissions.visible_orders.select("DISTINCT spree_orders.*").
+        complete.not_state(:canceled).
+        search(@params[:q])
```

Comment:
If we split the line, I'd keep all methods in their own line for consistency and readability, but that's my opinion.

Reply:
I have been doing that as well, I think it's easier to read.